src/requests/misc.jl

```diff
@@ -5,8 +5,19 @@ end
 function setTraceNotification_notification(params, server::LanguageServerInstance, conn)
 end
 
-function julia_getCurrentBlockRange_request(tdpp::TextDocumentPositionParams, server::LanguageServerInstance, conn)
-    doc = getdocument(server, URI2(tdpp.textDocument.uri))
+function julia_getCurrentBlockRange_request(tdpp::VersionedTextDocumentPositionParams, server::LanguageServerInstance, conn)
+    fallback = (Position(0, 0), Position(0, 0), tdpp.position)
+    uri = URI2(tdpp.textDocument.uri)
+
+    hasdocument(server, uri) || return fallback
```

Comment:
Same here, we should crash if the `doc` doesn't exist, something must have gone really wrong in that situation.